src/main/java/tech/ydb/spark/connector/YdbTable.java

```diff
@@ -291,8 +291,12 @@ public void deleteWhere(Filter[] filters) {
 
     @Override
     public boolean truncateTable() {
-        // TODO: implementation
-        throw new UnsupportedOperationException("Not supported yet.");
+        // todo: better ways to truncate YDB table?
```

Comment:
Truncate via delete will not work for a table of any significant size (probably starting from 100 MB of data). Proper implementation requires reading the table metadata, creating an empty table with similar structure and replacing the original table with the empty one.
Probably worth to keep unimplemented, instead of providing the non-working implementation.

Reply:
agree with that, will reconsider the impl

Reply:
> agree with that, will reconsider the impl

Unfortunately, that requires a significant amount of new code. The basic idea is the following:
1. grab the schema through DescribeTable,
2. convert it into the new CreateTableRequest, and execute it,
3. switch to the empty table using the RenameTable call.

Errors on step (3) should be handled with an attempt to drop the extra table created on step (2).

Reply:
written high-level skeleton in `YdbCatalog.truncateTable`, will give it one more iteration later to reduce missed meta info (indexes, constraints, params, etc.)